infrastructure/docker/traffic_router/run.sh

```diff
@@ -28,7 +28,8 @@
 # ORIGIN_URI # origin server (e.g. hotair), used to create a delivery service
 
 start() {
-	systemctl start traffic_router
+	chmod 777 starttr.sh
```

Comment:
this should probably just be done once, in the Dockerfile, not every time TR starts.

Reply:
Done in new commits.